### PR TITLE
Convert Activate Integration Entity Test

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that activates created integration entity

**Steps:**
1 - Login as Admin
2 - Navigate to System > Integration
3 - Add new integration
4 - Fill in all data according to data set
5 - Save &Activate the Integration
6 - Perform Assertions

Fixed Issues
#522 Convert ActivateIntegrationEntityTest to MFTF

### CODE REVIEW

File: app/code/Magento/Integration/Test/Mftf/ActionGroup/AssertAdminIntegrationResourcesPopupActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminIntegrationResourcesPopupActionGroup">
+        <see userInput="The integration you selected asks you to approve access to the following:" stepKey="seeContent"/>
```
Comment (rogyar) [inline app/code/Magento/Integration/Test/Mftf/ActionGroup/AssertAdminIntegrationResourcesPopupActionGroup.xml]: I would add and additional `waitForElementVisible` before `see` with the corresponding popup selector to make the assertion more robust.

File: app/code/Magento/Integration/Test/Mftf/ActionGroup/AssertAdminIntegrationTokensPopupActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminIntegrationTokensPopupActionGroup">
+        <see userInput="Integration Tokens for Extensions" stepKey="seeContent"/>
```
Comment (rogyar) [inline app/code/Magento/Integration/Test/Mftf/ActionGroup/AssertAdminIntegrationTokensPopupActionGroup.xml]: I would add and additional `waitForElementVisible` before `see` with the corresponding popup selector to make the assertion more robust.

File: app/code/Magento/Integration/Test/Mftf/ActionGroup/AssertAdminIntegrationResourcesPopupActionGroup.xml (hunk 1 above)
Comment (rogyar) [reply app/code/Magento/Integration/Test/Mftf/ActionGroup/AssertAdminIntegrationResourcesPopupActionGroup.xml]: Or specify a selector where we are looking for the text

**Comment by DmitryTsymbal** [comment]
Hello @rogyar 
Thank you for review!
I have added "waitForElementVisible" with popups selectors.
Please check.
Thanks

File: app/code/Magento/Integration/Test/Mftf/ActionGroup/AdminAllowResourcesAccessIntegrationActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/ActionGroup/AdminAllowResourcesAccessIntegrationActionGroup.xml]: Redundant space.

File: app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        <annotations>
+            <features value="Integration"/>
+            <stories value="Activation of System Integration"/>
+            <title value="Admin system integration activation"/>
+            <description value="Admin Activates Created Integration"/>
+            <group value="integration"/>
+            <group value="mtf_migrated"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="LoginAsAdmin"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml]: `AdminLoginActionGroup`

File: app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        </before>
+        <after>
+            <!-- Delete Integration -->
+            <actionGroup ref="AdminSearchIntegrationInGridActionGroup" stepKey="findCreatedIntegration">
+                <argument name="name" value="Test Integration"/>
+            </actionGroup>
+            <actionGroup ref="AdminDeleteIntegrationEntityActionGroup" stepKey="deleteIntegration"/>
+            <actionGroup ref="logout" stepKey="logout"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml]: `AdminLogoutActionGroup`

File: app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        </after>
+            <!-- Navigate To Integrations Page -->
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml]: Remove comments as these appear as single steps on Test results, even if you remove the real surrounding  steps.

File: app/code/Magento/Integration/Test/Mftf/ActionGroup/AdminAllowResourcesAccessIntegrationActionGroup.xml (hunk 3 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Integration/Test/Mftf/ActionGroup/AdminAllowResourcesAccessIntegrationActionGroup.xml]: Done

File: app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml (hunk 4 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml]: Done

File: app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml (hunk 5 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml]: Done

File: app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml (hunk 6 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Integration/Test/Mftf/Test/AdminActivateIntegrationEntityTest.xml]: Done

**Comment by lbajsarowicz** [comment]
@DmitryTsymbal The tests are failing.
Looks like the issue with Test Isolation.

Could you take a look at it?

**Pull request merged**